docs/docsite/rst/conf.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+"""Documentation Configuration."""
```

Comment:
FTR in such cases it's usually fine to put a docstring after the whole commented block